src/components/TitleWithDescription.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export const TitleWithDescription = (props) => {
+  const { title, description } = props;
+  return (
+    <>
+      <h3>{title}</h3>
+      <p>{description}</p>
+    </>
+  );
+};
```

Comment:
For curious readers :-D 

There is something called _Components dot notation_ that could be used here. 
I found some articles about:
short one: https://dev.to/alexandprivate/react-dot-notation-component-with-ts-49k8
a bit longer one: https://www.skovy.dev/blog/using-component-dot-notation-with-typescript-to-create-a-set-of-components?seed=5rb8hv

What you have here is definitely not wrong and for this context, it is fine. In larger apps, you would probably sooner or later have `TitleWithDescriptionAndLink` and `TitleWithDescriptionShort` and many other variants. Dot notation supports you to create `Title` and extend it with some relevant _blocks_ that can be freely extended (example of a block: `Title.Headline`, `Title.Description`, `Title.Link`

Moreover, it kind of enforce easy to ready structure.